Client/Renderer.cpp

```diff
@@ -87,6 +91,34 @@ GameTechRenderer::~GameTechRenderer()	{
     delete defaultShader;
 }
 
+
+
+void GameTechRenderer::RenderParticles()
+{
+    glEnable(GL_BLEND);
+    glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
+    BindShader(particleShader);
+
+    float screenAspect = (float)windowWidth / (float)windowHeight;
+    Matrix4 viewMatrix = gameWorld.GetMainCamera()->BuildViewMatrix();
+    Matrix4 projMatrix = gameWorld.GetMainCamera()->BuildProjectionMatrix(screenAspect);
+
+    int projLocation = glGetUniformLocation(particleShader->GetProgramID(), "projMatrix");
+    int viewLocation = glGetUniformLocation(particleShader->GetProgramID(), "viewMatrix");
+
+    glUniformMatrix4fv(projLocation, 1, false, (float*)&projMatrix);
+    glUniformMatrix4fv(viewLocation, 1, false, (float*)&viewMatrix);
+
+
+
+
+    for (auto ps : particleSystems)
```

Comment:
Needs a & in there to prevent the loop from creating a new ps for each iteration